madgraph/interface/reweight_interface.py

```diff
@@ -708,6 +760,235 @@ def do_launch(self, line):
         
         self.options['rwgt_name'] = None
 
+    def set_initial_mass_to_zero(self, event):
+
+        event[0].mass = 0.
+        event[1].mass = 0.
+        tot_E=0.
+        for ip,part in enumerate(event):
+            if ip >=2 :
```

Comment:
```suggestion
            if part.status == 1 :
```
not that the two are NOT equivalent if your event has resonances.